lib/containers/MultisigFollowUp.js

```diff
@@ -69,35 +71,58 @@ class MultisigFollowUp extends PureComponent {
     history.replace(`/${BASE_PATH}`);
   }
 
-  // TODO: styles and copy
   render() {
     const { joinKey, token, cosignerId } = this.props;
 
     return (
       <BoxGrid
         rowClass="text-center"
         colClass="flex-column justify-content-center m-auto"
-        colcount={1}
+        colcount={2}
       >
-        <Header type="h2">Multi Party Wallet</Header>
-        <Text>{this.headerCopy()}</Text>
-        {cosignerId && (
-          <Label text="Cosigner ID">
-            <Text>{cosignerId}</Text>
-          </Label>
-        )}
-        {joinKey && (
-          <Label text="Join Key">
-            <Text>{joinKey}</Text>
-          </Label>
-        )}
-        {token && (
-          <Label text="Cosigner Token">
-            <Text>{token}</Text>
-          </Label>
-        )}
-        <Text>Are you sure you wrote info down?</Text>
-        <Button onClick={() => this.complete()}>Yes</Button>
+        <div>
+          <Header type="h2">Multi Party Wallet</Header>
+          {cosignerId && (
+            <Label className="d-flex align-items-start" text="Cosigner ID">
+              <Text style={style.secretText} type="p">
+                {cosignerId}
+              </Text>
+            </Label>
+          )}
+          {joinKey && (
+            <Label className="d-flex align-items-start" text="Join Key">
+              <Text style={style.secretText} type="p">
+                {joinKey}
+              </Text>
+            </Label>
+          )}
+          {token && (
+            <Label className="d-flex align-items-start" text="Cosigner Token">
+              <Text style={style.secretText} type="p">
+                {token}
+              </Text>
+            </Label>
+          )}
+        </div>
+        <BoxGrid colcount={1} rowClass="d-inline">
+          <br />
+          <br /> {/* pad the top a bit */}
```

Comment:
fwiw there are some convenience classes in bootstrap to add padding and margin to items. Also support break points if not necessary at different widths (e.g. `pt-md-3` = padding-top-medium(and below)-3units)